lib/nerves_hub_link/certificate.ex

```diff
@@ -5,20 +5,6 @@ defmodule NervesHubLink.Certificate do
   @public_keys Application.get_env(:nerves_hub_link, :fwup_public_keys, [])
                |> NervesHubCLI.resolve_fwup_public_keys(org)
 
-  ca_cert_path =
-    System.get_env("NERVES_HUB_CA_CERTS") || Application.get_env(:nerves_hub_link, :ca_certs) ||
-      Path.join([File.cwd!(), "ssl", "prod"])
-
-  ca_certs =
-    ca_cert_path
-    |> File.ls!()
-    |> Enum.map(&File.read!(Path.join(ca_cert_path, &1)))
-    |> Enum.map(&X509.Certificate.to_der(X509.Certificate.from_pem!(&1)))
-
-  if ca_certs == [], do: raise("Couldn't find NervesHub CA certificates in '#{ca_cert_path}'")
```

Comment:
We should prob fail here during compilation if they are still using `:ca_certs`:

```elixir
if System.get_env("NERVES_HUB_CA_CERTS") || Application.get_env(:nerves_hub_link, :ca_certs) do
  raise("""
  Specifying `NERVES_HUB_CA_CERTS` environment variable or `config :nerves_hub_link, ca_certs: path`
  that is compiled into the module is no longer supported.

  If you are connecting to the public https://nerves-hub.org instance, simply remove env or config variable
  and the default certificates from NervesHubCAStore will be used by default.

  If you are connecting to your own instance with custom CA certificates, use the `:ca_store` config
  option to specific a module with a `cacerts/0` function or anonymous function that returns a list
  of DER encoded certificates:

    config :nerves_hub_link, ca_store: MyModule
      # OR
    config :nerves_hub_link, ca_store: &MyModule.get_cacerts/0

  If you have the certificates in DER format, you can also explicitly set them in the `:ssl` option:

    config :nerves_hub_link, ssl: [cacerts: my_der_list]
  """)
end
```